sound/soc/intel/boards/bxt_tdf8532.c

```diff
@@ -206,8 +206,8 @@ static struct snd_soc_dai_link broxton_tdf8532_dais[] = {
 		.no_pcm = 1,
 	},
 	{
-		/* SSP4 for vm */
-		.name = "vm_dai_link",
+		/* SSP4 for vm1 */
```

Comment:
What? Why would a VM be aware of which SSP is used for the amplifier? This makes no sense to me, what are you trying to do here?